### PR TITLE
[PATCH API-NEXT v2] api-next remove drv code

### PR DESCRIPTION
**Pull request by muvarov**
remove drv code from api-next which will be maintained in 2.0 branch to final worked version.

### CODE REVIEW

**Comment by lumag** [comment]
@muvarov I'd suggest to do a rebase instead of merge and to remove all drv headers in the first commit. Api-next has more headers than master.

**Comment by lumag** [comment]
@muvarov on the second thought your solution seems to be easier for the further master->api-next merges. Please ignore my previous comment.

File: doc/Doxyfile_common [hunk 1]
```diff
@@ -33,12 +33,9 @@ DOT_IMAGE_FORMAT = svg
...
-	     ODPDRV_ALIGNED(x)= \
 	     __BIG_ENDIAN_BITFIELD \
 	     __LITTLE_ENDIAN_BITFIELD \
 	     __x86_64__ \
 	     ODP_PACKED \
-	     ODPDRV_PACKED \
 	     ODP_DEPRECATE(x)=x \
-	     "ODP_HANDLE_T(type)=odp_handle_t type" \
-	     "ODPDRV_HANDLE_T(type)=odpdrv_handle_t type"
+	     "ODP_HANDLE_T(type)=odp_handle_t type"
```
Comment (lumag) [inline doc/Doxyfile_common]: @muvarov could you please port this part for master branch also?

Comment (muvarov) [reply doc/Doxyfile_common]: @lumag ok  I will stage to next branch changes.

Comment (muvarov) [reply doc/Doxyfile_common]: @lumag  actually that changes are not in master. So I will just not take them to next branch. So this chunk is only api-next specific.

**Comment by muvarov** [comment]
Merged. 2.0 branch has to revert drv remove changes and patch fd192cba

**Pull request merged**